RELEASE_NOTES.md

```diff
@@ -49,6 +49,10 @@ Foodcritic has been updated to 10.2.2\. This release includes the following new
 
 Kitchen-ec2 has been updated to 1.3.2 with support for Windows 2016 instances
 
+## Cookbook generator improvements
+
+Chef generate cookbook has been updated to better generate cookbooks for sharing with the Chef community. Generated cookbooks now require Chef client 12.1+, include the chef_version metadata, and use SPDX standard license strings.
```

Comment:
The `chef generate cookbook` command has been updated to improve sharing generated cookbooks with the Chef community.